src/createServer.js

```diff
@@ -1,3 +1,56 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable object-shorthand */
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  const server = http.createServer((req, res) => {
+    const normalizeUrl = new URL(req.url, `http://${req.headers.host}`);
+
+    const text = normalizeUrl.pathname.slice(1);
+    const getCase = normalizeUrl.searchParams.get('toCase');
+
+    if (!text || !getCase || !supportedCases.includes(getCase)) {
+      const errors = [];
+
+      if (!text) {
```

Comment:
Instead of checking for `!text` (and others) twice, just check for `errors.length !== 0` afterwards
```suggestion
      const errors = [];

      if (!text) {
```